src/mscorlib.WinRT-Facade/ref/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       uap;
       netcoreapp;
+      netstandard1.0;
+      netstandard1.2;
```

Comment:
Since ns1.0 and 1.2 are behaving the same you can eliminate 1.2.  You'll also need to add a `netstandard` config (without a version) or  `_netstandard` placeholder config due to the vertical build requirements.  I believe this will fix the CI failures.

Reply:
You'll need to add a nestandard config (without a version) due to the vertical build requirements.

Reply:
> the CI failures.

on my local machine, everything built fine using build -allConfigurations.

Reply:
It’s the netcoreapp build that’s failing (without -allconfigurations)